models/tmp/stg_zendesk__domain_name_tmp.sql

```diff
@@ -1,5 +1,5 @@
 --To disable this model, set the using_domain_names variable within your dbt_project.yml file to False.
 {{ config(enabled=var('using_domain_names', True)) }}
 
-select * 
-from {{ var('domain_name') }}
+select {{ dbt_utils.star(source('zendesk', 'domain_name')) }} 
+from {{ source('zendesk', 'domain_name') }} as domainname
```

Comment:
What was the need for the alias of the table here?

Reply:
Following up from my other comment in time_zone, should we just make the alias the default form for these tmp models?

Reply:
Following our chat in person, let's perform the alias on all tmp files. We will follow the format as defined below:
> Note: the `_table` to avoid the possibility of a field with the same name.
```sql
select {{ dbt_utils.star(source('zendesk', 'domain_name')) }} 
from {{ source('zendesk', 'domain_name') }} as domain_name_table
```